src/standalone/Soil/soil_hydrology_parameterizations.jl

```diff
@@ -7,7 +7,108 @@ export volumetric_liquid_fraction,
     impedance_factor,
     viscosity_factor,
     dψdϑ,
-    is_saturated
+    update_albedo!
+
+"""
+    albedo_from_moisture(surface_liquid_fraction::FT, albedo_dry::FT, albedo_wet::FT)
+
+Calculates pointwise albedo for any band as a function of volumetric soil water content given
+the dry and wet albedo values for that band. Implements eq.1 from Braghiere, R. K et al.
+"""
+function albedo_from_moisture(
+    surface_liquid_fraction::FT,
+    albedo_dry::FT,
+    albedo_wet::FT,
+) where {FT}
+    return albedo_dry * (1 - surface_liquid_fraction) +
+           albedo_wet * surface_liquid_fraction
+end
+
+
+"""
+    update_albedo!(bc::AtmosDrivenFluxBC, p, soil_domain, model_parameters)
+
+Calculates and updates PAR and NIR albedo as a function of volumetric soil water content at
+the top of the soil. If the soil layers are very large, the water content of the top layer
+is used in the calclulation
+"""
+function update_albedo!(bc::AtmosDrivenFluxBC, p, soil_domain, model_parameters)
+    (;
+        ν,
+        θ_r,
+        PAR_albedo_dry,
+        NIR_albedo_dry,
+        PAR_albedo_wet,
+        NIR_albedo_wet,
+        alebdo_calc_top_thickness,
+    ) = model_parameters
+    # ∫H_S_e_dz is the integral of effective saturation from (surface-alebdo_calc_top_thickness) to surface
+    ∫H_S_e_dz = p.soil.sfc_θ_l
+    FT = eltype(soil_domain.fields.Δz_top)
+    # checks if there is at least 1 layer centered within the top soil depth
+    if maximum(soil_domain.fields.Δz_top) < FT(0.5) * alebdo_calc_top_thickness
+        # ∫H_dz is integral of 1 from (surface-alebdo_calc_top_thickness) to surface
+        ∫H_dz = p.soil.sfc_scratch
+        # calculate depths of layer centers
+        @. p.soil.sub_sfc_scratch =
+            soil_domain.fields.z_sfc - soil_domain.fields.z
+        # zero all centers lower than boundry, set everything above to one
+        @. p.soil.sub_sfc_scratch =
```

Comment:
I think here it would be clearer to do:
```
        @. p.soil.sub_sfc_scratch =
            ClimaLand.heaviside.(
                FT(0.5) * alebdo_calc_top_thickness + sqrt(eps(FT)),
                soil_domain.fields.z_sfc - soil_domain.fields.z,
            )
```
vs:
```
        @. p.soil.sub_sfc_scratch =
            soil_domain.fields.z_sfc - soil_domain.fields.z
        # zero all centers lower than boundry, set everything above to one
        @. p.soil.sub_sfc_scratch =
            ClimaLand.heaviside.(
                FT(0.5) * alebdo_calc_top_thickness + sqrt(eps(FT)),
                p.soil.sub_sfc_scratch,
            )
```

Reply:
done